packages/cli/src/commands/org/create.ts

```diff
@@ -0,0 +1,50 @@
+import {Command, flags} from '@oclif/command';
+import {cli} from 'cli-ux';
+import {Config} from '../../lib/config/config';
+import {
+  Preconditions,
+  HasAccessToken,
+} from '../../lib/decorators/preconditions/';
+import {AuthenticatedClient} from '../../lib/platform/authenticatedClient';
+
+export default class OrgCreate extends Command {
+  static description = 'Create a new trial Coveo organization';
```

Comment:
Thinking long term, this could be for "creating any coveo organization", and it being a trial/dev/sandbox/prod could be controlled by a flag.

Otherwise, we will essentially always be stuck in that mode for this particular command (since it uses the "namespace" `org:create`) and will need to create a new major version to break this behaviour if we need to.

The default value for the flag could be `trial`, however. The net result will be something very similar to what you have now, but with more room to grow in the future.

Reply:
I think that yes, we should consider it a default value.
Also (tho this is probably a discussion that needs to be done 'live'), we should not be afraid to do breaking changes.
What's important is for the breaking change to be well thought, motivated and documented with great precision.

Also IMO, even if we'd have no flag today if we add flags and they are optional, if the behaviour with no flags stays the same, then there no breaking change.

The breaking change definition for the CLI IMO should only be based on the 'command typed in' to 'end-result relation exclusively.

Reply:
I also thought of that and I have heard that some folks in the platform are looking to simplify the LicenseTemplate/OrgTemplate mechanism, which is why i decided to not expose it. Like louis said, in the future, we could easily add an optional flag to expose that behavior.

Reply:
Fair enough !